R/plot.shapes.R

```diff
@@ -358,11 +358,34 @@ shape_noplot <- function(coords, v = NULL, params) {
 add_shape <- function(shape, clip = shape_noclip,
                       plot = shape_noplot,
                       parameters = list()) {
-  ## TODO
-  ## checkScalarString(shape)
-  ## checkFunction(clip)
-  ## checkFunction(plot)
-  ## checkList(parameters, named=TRUE)
+
+  if (!is.character(shape)) {
```

Comment:
Doesn't this check only that `shape` is a character, but not that it has length 1?

I don't feel very qualified to comment on such changes, since I'm not that knowledgeable about R 🙈 It's the sort of thing I'd ask you about @maelle 

Other than the above, nothing else stands out for me.

Reply:
thank you!